### PR TITLE
Add replication/HA demo to getting started guide.

### PR DESCRIPTION
**Pull request by tomwilkie**
These are the steps from the demos at KubeCon, showing basic operations, scalability and redundancy.

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -86,7 +100,70 @@ $ ./prometheus --config.file=./documentation/examples/prometheus.yml
...
+```
+
+In [the Grafana UI](http://localhost:3000) (username/password admin/admin), add a Prometheus datasource for Cortex (`http://cortex2:9009/api/prom`).
+
+## High availability with repication
+
+In this last demo we'll show how Cortex can replicate data among three nodes,
+and demonstrate Cortex can tolerate a node failure without affecting reads and writes.
+
+First clean up the last demo and run a new Consul and Grafana:
```
Comment (csmarchbanks) [inline docs/getting_started.md]: Perhaps add a note here saying you will still need a `cortex` network, and a Prometheus remote_write config pointed to localhost:9001? Skipping straight to this step does not work well right now.

File: docs/getting_started.md [hunk 2]
```diff
@@ -86,7 +100,70 @@ $ ./prometheus --config.file=./documentation/examples/prometheus.yml
...
+
+```sh
+$ docker rm -f cortex1 cortex2 consul grafana
+$ docker run -d --name=consul --network=cortex -e CONSUL_BIND_INTERFACE=eth0 consul
+$ docker run -d --name=grafana --network=cortex -p 3000:3000 grafana/grafana
+```
+
+Next update the `single-process-config.yaml` file to set replication factor to 3:
+
+```yaml
```
Comment (csmarchbanks) [inline docs/getting_started.md]: In previous demos (and this demo) we set the ring settings using command line arguments. This should be consistent (either a command line flag, or have the user edit the file everywhere).

File: docs/getting_started.md [hunk 3]
```diff
@@ -86,7 +100,70 @@ $ ./prometheus --config.file=./documentation/examples/prometheus.yml
 And Grafana at the second:
 
 ```sh
-$ docker run -d --network=cortex -p 3000:3000 grafana/grafana
+$ docker run -d --name=grafana --network=cortex -p 3000:3000 grafana/grafana
+```
+
+In [the Grafana UI](http://localhost:3000) (username/password admin/admin), add a Prometheus datasource for Cortex (`http://cortex2:9009/api/prom`).
+
+## High availability with repication
```
Comment (steven-sheehy) [inline docs/getting_started.md]:
```suggestion
## High availability with replication
```

**Comment by tomwilkie** [comment]
@csmarchbanks thanks for feedback!  PTAL

**Pull request merged**